src/test/resources/features/MFE/login-test.feature

```diff
@@ -0,0 +1,18 @@
+Feature: User Login check
+  I want to be able to see if a valid user login correctly
+
+ Background: User Opens visits login page
+    Given User opens the websites login page
+  Scenario Outline: User fill in credentials and login
+    
+    When User enters username "<username>"
```

Comment:
@k-joseph shared [this article](https://automationpanda.com/2017/01/30/bdd-101-writing-good-gherkin/) recently.  I suggest you take a look at it. The main upshot is that instead of describing every step the user takes, we want to describe *what they are doing* (i.e., the behaviour rather than the process).

```gherkin
Given user arrives at the websites login page
When the user logs in with "<username>" and "<password>"
   And the user selects location "<location>"
Then the user should arrive at the home page
```

I would suggest we also want to ensure we cover some known failure modes, e.g. if the user logs in with the incorrect username or password, what should happen? Should we have some scenarios around searching for the location or selecting alternative locations?

Reply:
I was trying to right all the steps found in the ticket description. Thank you. I'll implement it this way.

Sure I'll try adding the scenarios.Thank you very much Sir